sgid/index.html

```diff
@@ -0,0 +1,111 @@
+    </div>
+
+    <div class="grid__col section alternate">
+        <h2>SGID Index</h2>
+        <div class="grid__col--span-all grid__col--centered grid__col--s-hidden" style="width:568px">
+            <a href="{% link data/sgid-index/index.html %}">
+                <img alt="SGID Index Table Search" class="text-center" src="{% link images/sgid-index.png %}">
+            </a>
+        </div>
+        <p class="readable">The SGID Index is <b>the</b> master source of information about SGID datasets from AGRC and other GIS resources from many other state agencies. It is a great resource to quickly find GIS information important to you. The index lists the resource's category, name, and primary data steward, along with a short description of the resource and a link to a service endpoint (if available). </p>
+        <p class="section-read-more">
+            <a href="{% link data/sgid-index/index.html %}" class="btn">Search the Index</a>
+        </p>
+    </div>
+
+    <div class="grid__col section">
+        <h2>Open SGID</h2>
+        <p class="readable">The Open SGID is a publicly available database containing our curated collection of GIS datasets. You can use this vendor-neutral database connection to directly access Utah spatial Data in a wide variety of applications, including most GIS software. You can find the most up-to-date information about the Open SGID, including troubleshooting tips and an issue tracker, in the <a href="https://github.com/agrc/open-sgid">GitHub repository</a>. Please read, understand, and abide by the <a href="{% link about/policy/open-sgid/index.md %}">terms of use</a> prior to connecting.</p>
+        <p class="section-read-more">
+            <a href="{% link sgid/open-sgid/index.md %}" class="btn">Get Connected</a>
+        </p>
+    </div>
+
+    <div class="grid__col section alternate">
+        <h2>ArcGIS Online Hosted Feature Services</h2>
+        <div class="grid__col--span-all grid__col--centered grid__col--s-hidden" style="width:200px">
+            <a href="https://utah.maps.arcgis.com/home/groups.html">
+                <img alt="AGRC ArcGIS Online" class="text-center" src="{% link images/agol_location_group.png %}">
+            </a>
+        </div>
+        <p class="readable">AGRC uploads all the curated datasets to <a href="https://utah.maps.arcgis.com">ArcGIS Online</a> as hosted feature services. You can search for these layers in Esri products and add them to your projects. The name of every SGID item in ArcGIS Online will start with "Utah" (i.e., "Utah Roads") and have an AGRC-branded thumbnail. They are also shared with a Utah SGID group that matches their category.</p>
+        <p class="readable">Our ArcGIS Online organization also acts as a repository for "shelved" and "static" curated SGID datasets as described in our <a href="{% link about/policy/sgid/index.md%}"</a>SGID Database Policy</a>.</p>
```

Comment:
**suggestion**: possibly use the name of the agol org
**suggestion**: give a brief description that there are unique datasets not in the curated collection in agol somehow. 

please feel free to edit this but I think the intention is good.
```suggestion
        <p class="readable">The Utah ArcGIS Online organization also acts as a repository for "shelved" and "static" curated SGID datasets. These types of datasets are only available in ArcGIS Online and are described in our <a href="{% link about/policy/sgid/index.md%}"</a>SGID Database Policy</a>.</p>
```

Reply:
These edits look ok to you?

Reply:
yup, i'm good with that